src/alpaca_eval/leaderboards/data_AlpacaEval/alpaca_eval_gpt4_leaderboard.csv

```diff
@@ -1,101 +1,102 @@
 ,win_rate,standard_error,n_wins,n_wins_base,n_draws,n_total,mode,avg_length,discrete_win_rate,length_controlled_winrate
```

Comment:
Not sure what happened here, did you make any changes besides your model?

Reply:
I still completed the command, to add it to the loaderboard, it was like that. Maybe from rebasing or something?

Reply:
if it's the only change then it's fine, maybe it's due to sorting with alpaca eval LC